tests-hivemq-operator/src/integrationTest/java/com/hivemq/helmcharts/testcontainer/OperatorHelmChartContainer.java

```diff
@@ -147,10 +146,10 @@ public void upgradeLocalChart(final @NotNull String chartName, final @NotNull St
                 valuesFilePath);
 
         if (!execDeploy.getStderr().isEmpty()) {
-            // Shows also warnings
-            System.err.println(execDeploy.getStderr());
+            // shows also warnings
+            LOG.warn(execDeploy.getStderr());
         }
-        assertFalse(execDeploy.getStdout().isEmpty(), execDeploy.getStderr());
+        assertThat(execDeploy.getStdout().isEmpty()).describedAs(execDeploy::getStderr).isFalse();
```

Comment:
```suggestion
        assertThat(execDeploy.getStdout()).describedAs(execDeploy::getStderr).isNotEmpty();
```

Reply:
I guess, you forgot to commit your suggestion here? :)

Reply:
I used it just as a reminder, when scrolling through this after work. I prefer to fix this in IDEA and rebase it into the original commit.